packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -1340,7 +1340,7 @@ msgid "Main image"
 msgstr "Hlavní obrázek"
 
 msgid "Main page products"
-msgstr "Zboží na titulní stránce"
+msgstr "Propagované produkty"
```

Comment:
Is this change really needed in this PR?

Reply:
see https://github.com/shopsys/shopsys/pull/2066#discussion_r503730973

But when you mentioning it, it could be probably separated